src/commands/version.ts

```diff
@@ -8,7 +8,26 @@ export default (api: IApi) => {
     fn({ args }) {
       const version = require('../../package.json').version;
       if (!args.quiet) {
-        console.log(`father@${version}`);
+        const versions: string[] = [`father@${version}`];
+        // parts version
+        const parts: any = [
+          '@vercel/ncc',
+          'babel',
+          'esbuild',
+          'webpack',
+          'typescript',
+          '@microsoft/api-extractor',
+        ];
+
+        parts.forEach((part: string) => {
+          try {
+            // parts may not exist
+            versions.push(`${part}@${require(`${part}/package.json`).version}`);
+          } catch (error) {}
+        });
+        versions.forEach((version) => {
+          console.log(`Version: ${version}`);
```

Comment:
不用加颜色和 `logger.info`，感觉直接输出比较简洁 👀 

```suggestion
          console.log(version);
```